site/content/resources/upgrading.md

```diff
@@ -141,6 +141,66 @@ If your version of Contour is older than v1.18.2, please upgrade to v1.18.2 firs
     $ kubectl apply -f examples/contour/03-envoy.yaml
     ```
 
+## Upgrading Contour 1.18.2 to 1.18.3
```

Comment:
should we update the upgrade instructions after this as well? (currently 1.18.2 to 1.19.0)

Reply:
Yeah, I'll update. But this upgrade doc is increasingly terrifying to update, we may need to look at splitting it up somehow.